### PR TITLE
[libwebsockets] add libuv support

### PR DESCRIPTION
**Pull request by tsl0922**
Add libuv support to libwebsockets.

I'm trying to build [ttyd](https://github.com/tsl0922/ttyd) with vcpkg, requires libuv support for libwebsockets.

### CODE REVIEW

File: ports/libwebsockets/portfile.cmake [hunk 1]
```diff
@@ -141,6 +141,7 @@ vcpkg_configure_cmake(
         -DLWS_IPV6=ON
```
Comment (LilyWangL) [inline ports/libwebsockets/portfile.cmake]: `include(vcpkg_common_functions)` is deprecated. Please remove it.

**Review by LilyWangL — CHANGES_REQUESTED** [review]

File: ports/libwebsockets/portfile.cmake (hunk 1 above)
Comment (tsl0922) [reply ports/libwebsockets/portfile.cmake]: Done

**Review by LilyWangL — APPROVED** [review]

**Comment by ras0219-msft** [comment]
The linux breaks are unrelated and should not block merging this PR

**Comment by ras0219-msft** [comment]
Thanks for the PR!

**Pull request merged**